src/getHumanAge.test.js

```diff
@@ -3,8 +3,92 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
```

Comment:
try to add 1-2 tests where 2 values in the output will be different (about the .toEqual)